docs/guides/key-manager/upgrade-lsp6.md

```diff
@@ -0,0 +1,244 @@
+} from './constants.js';
+import Web3 from 'web3';
+const web3 = new Web3('https://rpc.l14.lukso.network');
+```
+
+## Step 3 - Initialize the controller account
+
+In order to use the controller account you need to initialize it using web3.js.
+
+```js
+const account = web3.eth.accounts.privateKeyToAccount(PRIVATE_KEY);
+web3.eth.accounts.wallet.add(account);
+```
+
+## Step 4 - Initialize the old LSP6 Key Manager
+
+In order to transfer ownership of your Universal Profile you need to initialize your current LSP6 Key Manager.
+
+```js
+const oldKeyManager = new web3.eth.Contract(
+  LSP6KeyManager.abi,
+  keyManagerAddress,
+);
+```
+
+## Step 5 - Deploy the new LSP6 Key Manager
+
+Deploy a new LSP6 Key Manager with the latest updates.
+
+```js
+const NewKeyManager = new web3.eth.Contract(LSP6KeyManager.abi);
+const newKeyManager = await NewKeyManager.deploy({
+  data: LSP6KeyManager.bytecode,
+  arguments: [universalProfileAddress],
+}).send({
+  from: account.address,
+  gas: 3000000,
+  gasPrice: '1000000000',
+});
+
+console.log(newKeyManager._address);
+```
+
+## Step 6 - Transfer ownership of the Universal Profile
+
+Create a payload for `transferOwnership(address)` and tarnsfer the ownership of your Universal Profile from your current LSP6 Key Manager.
+
+```js
+const transferOwnershipPayload =
+  web3.eth.abi.encodeFunctionSignature('transferOwnership(address)') +
+  web3.eth.abi.encodeParameter('address', newKeyManager._address).substring(2);
+
+await oldKeyManager.methods['execute(bytes)'](transferOwnershipPayload).send({
+  from: account.address,
+  gas: 1000000,
+  gasPrice: '1000000000',
+});
+```
+
+## Step 7 - Claim ownership of the Universal Profile
+
+Create a payload for `claimOwnership()` and claim the ownership of your Universal Profile from your new LSP6 Key Manager.
+
+```js
+const claimOwnershipPayload =
+  web3.eth.abi.encodeFunctionSignature('claimOwnership()');
+
+await newKeyManager.methods['execute(bytes)'](claimOwnershipPayload).send({
+  from: account.address,
+  gas: 1000000,
+  gasPrice: '1000000000',
+});
+```
+
+## Final code
+
+```javascript title="constants.js"
+export const PRIVATE_KEY = '0x...';
+export const universalProfileAddress = '0x...';
+export const keyManagerAddress = '0x...';
+```
+
+```javascript title="upgrade-lsp6.js"
+import LSP6KeyManager from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json' assert { type: 'json' };
+import {
+  PRIVATE_KEY,
+  keyManagerAddress,
+  universalProfileAddress,
+} from './constants.js';
+import Web3 from 'web3';
+const web3 = new Web3('https://rpc.l14.lukso.network');
+
+const upgradeLSP6 = async () => {
+  // Initialize the controller account
+  const account = web3.eth.accounts.privateKeyToAccount(PRIVATE_KEY);
+  web3.eth.accounts.wallet.add(account);
+
+  // Initialize your current LSP6 Key Manager
+  const oldKeyManager = new web3.eth.Contract(
+    LSP6KeyManager.abi,
+    keyManagerAddress,
+  );
+
+  // Deploy a new LSP6 Key Manager
```

Comment:
@CJ42 whats the right convention? Is it LSP6 Key Manager or LSP6 KeyManager? Just worried myself, nothing to judge here.

Reply:
I saw in other guides it was separate so I kept it that way

Reply:
Ok, great- so I know how to write standard names from now on 👍🏻